lib/vimeo.js

```diff
@@ -326,7 +326,7 @@ Vimeo.prototype.accessToken = function (code, redirectUri, fn) {
  *
  * @param  {string} redirectUri   The URI that will exchange a code for an access token. Must match
  *                                the URI in your API app settings.
- * @param  {string} scope         An array of scopes. See https://developer.vimeo.com/api/authentication#scopes
+ * @param  {string} scope         A space-separated list of scopes. See https://developer.vimeo.com/api/authentication#scopes
```

Comment:
An array works, but the type is `string` so I changed the description. We could also change the type to `string|string[]`.

Reply:
Let's change it to `string|string[]`.